test/utils_test.exs

```diff
@@ -68,7 +68,7 @@ defmodule UtilsTest do
   @tag timeout: 120000 # 120s
   test "can build a release and boot it up" do
     with_app do
-      #capture_io(fn ->
+      capture_io(fn ->
```

Comment:
I actually have this commented out since when it fails it's impossible to see why. Could you instead remove the `capture_io` wrapper entirely? No sense in having it anymore IMO.